Trappist/src/Promact.Trappist.Core/Controllers/TestController.cs

```diff
@@ -9,12 +9,12 @@ namespace Promact.Trappist.Core.Controllers
     [Route("api/tests")]
     public class TestsController : Controller
     {
-        private ITestsRepository _test;
-       private IStringConstants _stringConstant;
-            
+        private readonly ITestsRepository _testRepository;
+        private readonly IStringConstants _stringConstant;
+  
         public TestsController(ITestsRepository test,IStringConstants stringConstant)
         {
-            _test = test;
+            _testRepository = test;
```

Comment:
do i need to tell seperately that, also rename this test as testRepository name?